src/types.ts

```diff
@@ -492,6 +492,9 @@ interface IntegratorShareFeeConfig extends GaslessFeeConfigBase {
     integratorSharePercentage: BigNumber;
 }
 
+export type IntegratorFeeConfig = VolumeBasedFeeConfig;
+export type ZeroExFeeConfig = VolumeBasedFeeConfig | IntegratorShareFeeConfig;
```

Comment:
Hmmm, these type aliases feel like they might not be all that helpful. My vote would be to remove them since you can always just declare stuff like:


```
interface MyInterface {
  zeroExFeeConfig: VolumeBasedFeeConfig | IntegratorShareFeeConfig;
  integratorFeeConfig: VolumeBasedFeeConfig;
}
```

instead of 

```
interface MyInterface {
  zeroExFeeConfig: ZeroExFeeConfig;
  integratorFeeConfig: IntegratorFeeConfig;
}
```
I prefer not creating extra types